js/net_test.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
 
 import * as deno from "deno";
-import { testPerm, assert, assertEqual, deferred } from "./test_util.ts";
+import { testPerm, assert, assertEqual } from "./test_util.ts";
+import { deferred } from "./util.ts";
```

Comment:
Cool! I forgot about this when I suggested it - but this is only newly possible because of your work in #859 
`js/net_test.ts` runs inside of Deno, but `js/util.ts` runs inside of Node/rollup. But now we are able to seamlessly load `util.ts` in Deno too.